test.js

```diff
@@ -0,0 +1,21 @@
+// declare two variables that has module scope
```

Comment:
Test file! I love how you are writing a small file to learn Javascript!

Of course, this does not belong to our app. 😄

Reply:
Oops I usually try to go back to the assignments given in the past from time to time in case I forget but it looks like I forgot to delete it afterwards haha thank you for catching that!

Reply:
After you delete the file, you can commit and push to your branch.